src/nodes.coffee

```diff
@@ -3553,6 +3646,9 @@ exports.Parens = class Parens extends Base
     return @wrapInBraces fragments if @csxAttribute
     if bare then fragments else @wrapInParentheses fragments
 
+  ast: ->
```

Comment:
Parens don't generate AST nodes. It looks like Babel includes some paren-related info under `extra` (eg `extra: {parenthesized: true, parenStart: <range start>}`) of the parenthesized node, but `espree` doesn't (ESLint examines the source code/tokens for paren-related stuff). I haven't run into a compelling reason to try and include something like the Babel-style `parenthesized` in the AST yet. But it's something we may eventually want to do